src/edu/pugetsound/mathcs/architecture_nlp/features/spf/IInteract.java

```diff
@@ -0,0 +1,15 @@
+package edu.pugetsound.mathcs.architecture_nlp.features.spf;
+
+import edu.cornell.cs.nlp.spf.data.IDataItem;
+import edu.cornell.cs.nlp.spf.parser.IDerivation;
+import edu.cornell.cs.nlp.spf.parser.ccg.model.IModelImmutable;
+
```

Comment:
Add a Javadoc comment to this interface explaining the purpose of the interface. Include an @author tag with your name